odxtools/cli/main.py

```diff
@@ -47,11 +53,6 @@ def start_cli():
         argparser.print_usage()
         exit()
 
-    if args.subparser_name == "list":
-        list.run(args)
-    elif args.subparser_name == "browse":
-        browse.run(args)
-    elif args.subparser_name == "snoop":
-        snoop.run(args)
-    elif args.subparser_name == "find":
-        find.run(args)
+    for tool in tool_modules:
+        if tool.__name__ == args.subparser_name:
```

Comment:
```suggestion
        if tool.__name__.endswith(args.subparser_name):
```

Reply:
I guess I want this to be a bit more bulletproof, IOW, I don't want to rely on this not breaking in the future.